storage-proofs-update/src/constants.rs

```diff
@@ -0,0 +1,112 @@
+use filecoin_hashers::{
+    poseidon::{PoseidonDomain, PoseidonHasher},
+    sha256::{Sha256Domain, Sha256Hasher},
+};
+use generic_array::typenum::{Unsigned, U0, U2, U8};
+use storage_proofs_core::merkle::{BinaryMerkleTree, LCStore, LCTree, MerkleTreeTrait};
+
+// Sector-sizes measured in nodes.
+pub const SECTOR_SIZE_1_KIB: usize = 1 << 5;
```

Comment:
Can we share these constants with `filecoin-proofs`?

Reply:
if we need them in here, maybe export them and import them in filecoin-proofs, not sure